stb_vorbis.c

```diff
@@ -904,6 +905,7 @@ static int error(vorb *f, enum STBVorbisError e)
    return 0;
 }
 
+#define UNUSED_PARAMETER(x) (void)(x)
```

Comment:
This is a matter of opinion, but is defining `UNUSED_PARAMETER` necessary, as most people know what `(void)foo` does? I could see the use case of searching for the macro to find dead parameters, but only in cases where it's an actual dead parameter and not something that is compiled conditionally.